src/Gui/MainWindow.cpp

```diff
@@ -1276,9 +1276,7 @@ void MainWindow::delayedStartup()
     ParameterGrp::handle hGrp = WindowParameter::getDefaultParameter()->GetGroup("Document");
     if (hGrp->GetBool("CreateNewDoc", false)) {
         if (App::GetApplication().getDocuments().size()==0){
-            App::GetApplication().newDocument();
-            Gui::Command::doCommand(Gui::Command::Gui,
-                "Gui.activeDocument().activeView().viewDefaultOrientation()");
+            Application::Instance->commandManager().runCommandByName("Std_New");
```

Comment:
@wwmayer is there any risk in running a Gui command here?

Reply:
Should be fine.